create-rust-app/src/auth/mod.rs

```diff
@@ -15,7 +15,7 @@ pub mod controller;
 mod endpoints;
 pub use endpoints::*;
 
-mod mail;
+pub(crate) mod mail;
```

Comment:
allow these default templates to be accessed within the crate (needed for the DefaultMailTemplate struct which is defined in mailer.rs, which is outside of the scope of the auth module